CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 * Your contribution here.
 * [#202](https://github.com/ruby-grape/grape-entity/pull/202): Fix: Reset `@using_class` memoization on `.setup` - [@rngtng](https://github.com/rngtng).
+* [#203](https://github.com/ruby-grape/grape-entity/pull/203): Fix: return `self` instead of `nil` if no attribute was deleted - [@rngtng](https://github.com/rngtng).
```

Comment:
This seems incorrect, shouldn't it say "`Grape::Entity::Exposure::NestingExposure::NestedExposures.delete_if` always
 returns exposures."? It's unlcear what `self` is and what the story here is ;)